src/Command.php

```diff
@@ -6,203 +6,188 @@
 
 namespace ZF\ComposerAutoloading;
 
+use Zend\Stdlib\ConsoleHelper;
+
 class Command
 {
-    /**
-     * @var string Composer binary name/location
-     */
-    private $composer;
+    const DEFAULT_COMMAND_NAME = 'zf-composer-autoloading';
 
     /**
-     * Default argument values.
-     *
-     * @param array
+     * @var string
      */
-    private $defaults = [
-        'composer' => 'composer',
-        'type' => false,
-        'modulesPath' => 'module',
-    ];
+    private $projectDir = '.';
 
     /**
-     * @var string Path to composer.json
+     * @var string[]
      */
-    private $composerJsonFile;
+    private $commands = [
+        'disable' => Command\Disable::class,
+        'enable' => Command\Enable::class,
+    ];
 
     /**
-     * @var array Parsed composer.json contents
+     * @var array
      */
-    private $composerPackage;
+    private $helpArgs = ['--help', '-h', 'help'];
 
     /**
-     * @var string Module name
+     * @var string Composer binary name/location
      */
-    private $module;
+    private $composer = 'composer';
 
     /**
-     * @var string Filesystem path to module
+     * @var string One of psr-0 or psr-4
      */
-    private $modulePath;
+    private $type;
 
     /**
      * @var string Filesystem path to modules directory
      */
-    private $modulesPath;
+    private $modulesPath = 'module';
 
     /**
-     * @var string Working path
+     * @var string Module name
      */
-    private $path;
+    private $module;
 
     /**
-     * @var string One of psr-0 or psr-4
+     * @var string Filesystem path to module
      */
-    private $type;
+    private $modulePath;
 
     /**
-     * @var string $path
+     * @param string $command
+     * @param null|ConsoleHelper $console
      */
-    public function __construct($path)
+    public function __construct($command = self::DEFAULT_COMMAND_NAME, ConsoleHelper $console = null)
     {
-        $this->path = $path;
-        $this->composerJsonFile = sprintf('%s/composer.json', $path);
+        $this->command = (string)$command;
```

Comment:
Yeh, there should be space. It is not in version 1.0.0 of zend-coding-standard unfortunately but it is coming with big upgrade ;-)